src/server.js

```diff
@@ -0,0 +1,36 @@
+// set up express
```

Comment:
please pop this file (and all other files/folders related to the server) inside of a server folder in the root directory